include/gazebo_lidar_plugin.h

```diff
@@ -63,7 +64,7 @@ namespace gazebo
       std::string namespace_;
       double min_distance_;
       double max_distance_;
-
+      int facing_;
```

Comment:
Let's call this rotation instead of facing.

Reply:
Done.